base_layer/wallet/src/output_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -73,12 +73,55 @@ pub struct OutputManagerSqliteDatabase {
 
 impl OutputManagerSqliteDatabase {
     pub fn new(database_connection: WalletDbConnection, cipher: Option<Aes256Gcm>) -> Self {
+        // TODO: Remove this migration for the next testnet reset (currently on Dibbler) #testnet_reset
+        let _ = OutputManagerSqliteDatabase::update_json_output_features_to_v1(database_connection.clone());
+
         Self {
             database_connection,
             cipher: Arc::new(RwLock::new(cipher)),
         }
     }
 
+    // TODO: Remove this migration for the next testnet reset (currently on Dibbler) #testnet_reset
+    fn update_json_output_features_to_v1(
```

Comment:
This should not be done IMO. Newly created outputs should get the V1 output features only if they are created with output v1 or higher